readit/database.py

```diff
@@ -182,10 +190,9 @@ def search_url(self, search_value):
                 self.cursor.execute(
                     """ SELECT id, url, tags, date, time
                                     FROM bookmarks WHERE tags=?""",
-                    (self.search,),
+                    (self.search),
                 )
```

Comment:
@amolkahat 
We search a group of URLs by either tag name or URL substring.
I think, we need to keep ```(self.search,)``` as it is. Otherwise, it throws below error. 
Please check this error.

```
(n) [ghubale@localhost readit]$ readit -s mytag

Traceback (most recent call last):
  File "/home/ghubale/readit/readit/readit/database.py", line 193, in search_url
    (self.search),
sqlite3.ProgrammingError: Incorrect number of bindings supplied. The current statement uses 1, and there are 5 supplied.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ghubale/readit/n/bin/readit", line 11, in <module>
    load_entry_point('readit', 'console_scripts', 'readit')()
  File "/home/ghubale/readit/n/lib64/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/ghubale/readit/n/lib64/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/ghubale/readit/n/lib64/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ghubale/readit/n/lib64/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/ghubale/readit/readit/readit/cli.py", line 130, in main
    output.print_bookmarks(database_connection.search_url(search))
  File "/home/ghubale/readit/readit/readit/database.py", line 208, in search_url
    raise sqlite3.OperationalError
sqlite3.OperationalError
```

Reply:
strange...  i'm not able to reproduce this error.

Reply:
I can still able to reproduce this issue with python3.7.5
I created new issue for it - #159